projects/packages/forms/package.json

```diff
@@ -82,6 +81,7 @@
 		"@babel/core": "7.27.7",
 		"@babel/runtime": "7.27.6",
 		"@testing-library/dom": "10.4.0",
+		"@testing-library/react": "^14.2.1",
```

Comment:
I'll update this version to match:
- https://github.com/Automattic/jetpack/pull/44217